framework.go

```diff
@@ -22,7 +22,9 @@ import (
 )
 
 const (
-	KubeconfigEnvVar = "E2E_KUBECONFIG"
+	EnvKubeconfig               = "E2E_KUBECONFIG"
```

Comment:
The reason I went for `Env<ConstName>` is so that it shows up neatly in your editor's suggestions when you just time "Env"

Reply:
Love this! Nice attention to detail. :)